### PR TITLE
change version scheme

### PR DESCRIPTION
**Pull request by aethelwulffe**
Added a 0.0.1 to 1.0.0 upgrade that takes the good stuff from 420-431,
then adds the table name changes at the end.  Changed the version of the
application and database to 1.  Removed the clickoptions file and
conversion, as that is a much earlier version change.

### CODE REVIEW

**Comment by aethelwulffe** [comment]
Issue #47

File: sql_upgrade.php [hunk 1]
```diff
@@ -110,7 +110,7 @@
 foreach ($versions as $version => $filename) {
   echo " <option value='$version'";
   // Defaulting to most recent version, which is now 4.2.1.
```
Comment (tmccormi) [inline sql_upgrade.php]: bad comment, refers to old version #.  may change comment to say forked from version 4.2.1 or drop it

File: version.php [hunk 2]
```diff
@@ -17,16 +17,14 @@
 // is a database change in the course of development.  It is used
 // internally to determine when a database upgrade is needed.
 //
-$v_database = 164;
+$v_database = 1;
```
Comment (tmccormi) [inline version.php]: Has database.sql been changed to be version 1 in the table?

**Comment by aethelwulffe** [comment]
database.sql does the following:
INSERT INTO version (v_major, v_minor, v_patch, v_realpatch, v_tag, v_database, v_acl) VALUES (0, 0, 0, 0, '', 0, 0);

**Comment by aethelwulffe** [comment]
Changed version comment.

**Pull request merged**